sw/airborne/modules/gpio_ext/gpio_ext_common.c

```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright (C) Tom van Dijk <tomvand@users.noreply.github.com>
+ *
+ * This file is part of paparazzi
+ *
+ * paparazzi is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2, or (at your option)
+ * any later version.
+ *
+ * paparazzi is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with paparazzi; see the file COPYING.  If not, see
+ * <http://www.gnu.org/licenses/>.
+ */
+
+/** @file "modules/gpio_ext/gpio_ext_common.c"
+ * @author Tom van Dijk <tomvand@users.noreply.github.com>
+ * Common external GPIO functions.
+ */
+
+#include "gpio_ext_common.h"
+
+#include "generated/airframe.h"
+#include "mcu_periph/gpio.h"
+#include "led.h"
+
+#include <stdint.h>
+#include <stdbool.h>
+#include <assert.h>
+
+
+/* External GPIO providers */
+#include "gpio_ext_pca95xx.h"
+
+#define GPIO_EXT_NOT_PROVIDED NULL
+#define GPIO_EXT_PCA95XX &pca95xx_functions
```

Comment:
doesn't seems to be used, and should not be here anyway I think

Reply:
This define is used in the airframe file, to specify which module provides the GPIOEXTx port. I can move it to gpio_ext_pca95xx.h instead?

Reply:
yes, if possible

Reply:
fixed